### ISSUE TITLE
How can I cancel dragging into iframe?

### ISSUE THREAD

**Issue by CarlosBalladares** [prevwong/craft.js#445]
when the canvas is inside an iframe and the draggable html node is outside that iframe im unable to cancel the dragging action

**Comment by trionfordring** [comment]
I had the same problem.

**Comment by Andy-lm** [comment]
I had the same problem.
```
<iframe
      ref={(dom: HTMLIFrameElement) => connect(dom)}
    >
      {children}
</iframe>
```

**Comment by AstralBaby** [comment]
did you fix that?

**Comment by CarlosBalladares** [comment]
No, its been a long time and my project that used the iframe to isolate the styles of  the component still has that behaviour. Some workarounds, if you are using iframe to isolate styles like I was, consider using a shadow dom instead or making a simple drag and drop with another library like react dnd kit .
